src/data/help.csv

```diff
@@ -0,0 +1,83 @@
+"text","translation"
```

Comment:
if you could add size markup for longer text that would be great
(https://user-images.githubusercontent.com/80930594/132095682-5ded8547-4fea-4097-b8cb-95aa5bdb5ff0.png)
(https://user-images.githubusercontent.com/80930594/132095627-960a0f93-9877-4f75-80db-8e173a8e72c3.png)
(https://user-images.githubusercontent.com/80930594/132095619-da948a1c-943c-4853-a7ca-7ac8483513eb.png)

Reply:
some lines with color markups missing closing tags, a lot...